cli/cmd/plugin/ui/web/tanzu-ui/src/components/DockerForm.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ChangeEvent, useContext } from "react";
+
+import { CdsInput } from "@cds/react/input";
+import { CdsFormGroup } from "@cds/react/forms";
+import { CdsButton } from "@cds/react/button";
+
+import { TEXT_CHANGE } from "../constants/actionTypes";
+import { Store } from "../stores/store";
+
+
+function DockerForm () {
```

Comment:
Could we change this to something like vSphereAuthForm since the content consists of vsphere auth steps.